test/server/config_validation/xds_fuzz.cc

```diff
@@ -61,13 +62,14 @@ XdsFuzzTest::XdsFuzzTest(const test::server::config_validation::XdsTestCase& inp
                                          ? "GRPC"
                                          : "DELTA_GRPC",
                                      api_version)),
-      actions_(input.actions()), version_(1), api_version_(api_version),
-      ip_version_(TestEnvironment::getIpVersionsForTest()[0]), num_added_(0), num_modified_(0),
-      num_removed_(0) {
+      verifier_(input.config().sotw_or_delta()), actions_(input.actions()), version_(1),
+      api_version_(api_version), ip_version_(TestEnvironment::getIpVersionsForTest()[0]) {
   use_lds_ = false;
   create_xds_upstream_ = true;
   tls_xds_upstream_ = false;
 
+  drain_time_ = std::chrono::seconds(5);
```

Comment:
Does this have some real-time implication? One thing we worry about in CI is what happens if CI is extra slow and we flake.

Reply:
I was just putting this in to try and test draining with simulated time, I don't actually use it yet (see issue https://github.com/envoyproxy/envoy/issues/12182).  I can remove it for now.

Reply:
Yeah, best to remove unrelated stuff.